### DISCUSSION TITLE
Fix "Missing route to pin" warning

### DISCUSSION THREAD

**Discussion by lustefan97** [The-OpenROAD-Project/OpenROAD#4317] [Q&A]
Hello,

I am trying to run a custom script to characterize OpenROAD's behavior when the design utilization is varying for a fixed placement_density, using the ASAP7 PDK, when all IO's are on the top metal layer.

I currently use the ibex design as a test design and I am aiming at a DU of 60%, with a placement density fixed at 1. The issue I have is that I encounter this warning during the global routing step : 

"[WARNING GRT-0026] Missing route to pin (name of the pin)"

It does this for all pins. During detailed routing it seems to try and fix this problem (as OpenROAD give me the [INFO-DRT-1000] code), but it fails to do so and thus stop the flow after indicating : 

"[ERROR DRT-0218] Guide is not connected to design"

I want to fix this issue, but I have no idea as to where to look, or what to do...

I use the "grid-strategy-M1-M2-M5-M6.tcl" as well as the make_tracks.tcl found in the OpenROAD-flow-scripts/flow/platforms/asap7/ directory.

**Comment by maliberty** [comment]
@eder-matheus is this warning one that you fixed recently?

@lustefan97 its better to give an issue with a test case if possible.

**Reply by eder-matheus** [reply]
Yes, this is the one @arthurjolo fixed recently. But that is a different problem because the previous warnings weren't leading to DRT errors.

@lustefan97 A reproducible test case would be really nice.

**Reply by lustefan97** [reply]
@maliberty @eder-matheus I asked this question here because I thought that it may be a mistake on my part that I could fix (although I managed to PnR that exact same design before), I'll open an issue with a test case !
